views/__init__.py

```diff
@@ -180,6 +180,11 @@ def login():
         return jsonify(success="Authenticated"), 200
 
 # 
+@app.route('/login', methods=['GET'])
+def login_form():
+    return redirect('/#login')
```

Comment:
So we need an if statement here and below as the one here: https://github.com/phenopolis/phenopolis/blob/master/views/__init__.py#L187